feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -92,10 +93,16 @@ func configureDeviceId(t *testing.T, dut *ondatra.DUTDevice) {
 	gnmi.Replace(t, dut, gnmi.OC().Component(p4rtNode).Config(), &c)
 }
 
-// configurePortId configures p4rt port-id on the DUT.
+// configurePortId configures p4rt port-id and interface type on the DUT.
 func configurePortId(t *testing.T, dut *ondatra.DUTDevice) {
+	d := gnmi.OC()
 	portName := dut.Port(t, "port1").Name()
-	gnmi.Replace(t, dut, gnmi.OC().Interface(portName).Id().Config(), portId)
+	curr_intf := &oc.Interface{Name: ygot.String(portName),
+		Type: if_type}
+	gnmi.Replace(t, dut, d.Interface(portName).Config(), curr_intf)
+
+	gnmi.Replace(t, dut, d.Interface(portName).Id().Config(), portId)
```

Comment:
this gnmi.Replace is not needed. Instead add `Id` as part of the `curIntf` struct.

Reply:
I have removed the extra line.